lib/atlassian-jwt-authentication/filters.rb

```diff
@@ -130,7 +130,7 @@ def _verify_jwt(addon_key, consider_param = false, skip_qsh_verification: false)
         jwt = possible_jwt if algorithm == 'JWT'
       end
 
-      jwt_verification = AtlassianJwtAuthentication::JWTVerification.new(addon_key, jwt, request)
+      jwt_verification = AtlassianJwtAuthentication::JWTVerification.new(addon_key, nil, jwt, request)
```

Comment:
OK, but does it mean know that the default use will always fail, treat JWT as invalid?

Reply:
nope:

```
    def verify_aud
      return unless (options_aud = @options[:aud])  # <-- @options[:aud] will be nil, so this whole line will return prematurely

      aud = @payload['aud']
      raise(JWT::InvalidAudError, "Invalid audience. Expected #{options_aud}, received #{aud || '<none>'}") if ([*aud] & [*options_aud]).empty?
    end
```